docs/INTEGRATION_VARIANTS.md

```diff
@@ -0,0 +1,259 @@
+## Variants
+
+Configure variants by setting the variant field in either:
+- Controller config: `controllers[controller].config.settings.variants.field`
+- Recommendation config: `instantiators.recommendation.config.settings.variants.field`
+
+Example -
+
+```typescript
+const config = {
+	instantiators: {
+		recommendation: {
+			components: {
+				Bundle: async () => (await import('./components/Recommendations/Bundle/Bundle')).Bundle,
+			},
+			config: {
+				settings: {
+					variants: {
+						field: 'ss_variants',
+					},
+				},
+			},
+		},
+	},
+	controllers: {
+		search: [
+			{
+				config: {
+					id: 'search',
+					settings: {
+						variants: {
+							field: 'ss_variants',
+						},
+					},
+				},
+			},
+		],
+	},
+};
+
+const snap = new Snap(config);
+```
+
+Once configured, each result in `controller.store.results` includes a `variants` object with:
+
+**Properties:**
+- `active` - Currently selected variant data
+- `data` - Array of all available variants
+- `selections` - Variant selection state and options (detailed below)
+
+**Methods:**
+- `setActive(variant)` - Select a specific variant
+- `update(newData)` - Update variant data post-initialization
+- `refineSelections()` - Filter available options to prevent invalid combinations
+- `makeSelections()` - *Internal use only*
```

Comment:
Since other properties were left out from above... maybe just leave this (and maybe `refineSelections`) out? These have no use to the integrator do they?